library/src/main/java/com/eggheadgames/aboutbox/activity/AboutActivity.java

```diff
@@ -23,6 +23,7 @@
 
     public static void launch(Activity activity) {
         Intent intent = new Intent(activity, AboutActivity.class);
+        intent.setFlags(Intent.FLAG_ACTIVITY_REORDER_TO_FRONT);
```

Comment:
This fixes a bug with the way the intent/activity is resumed after being placed in the background.

My activity looks like this:

![screenshot_20170824-114950](https://user-images.githubusercontent.com/705618/29722257-f160ff1c-8974-11e7-8de1-3ef4002574a3.png)

After being in the background for a while, and resumed, it looks like this:

![screenshot_20170824-113935](https://user-images.githubusercontent.com/705618/29722279-028afef0-8975-11e7-903f-f39aed0d02ef.png)

Since the activity/intent was never actually placed to the front. when resumed a new non-initialized is created, hence the issue.

Reply:
Good catch! It's on my list to review this PR and merge this weekend. Thanks again!

Reply:
@ethauvin Haven't forgotten about this. Just waiting for someone else to also check this new `FLAG_ACTIVITY_REORDER_TO_FRONT` fix.  If you want to move that part to a separate PR, I'll approve the rest of this one. Probably just another day though, so up to you.

@AlexDibrivnyy As discussed, when you get a chance, can you take a closer look at this. Thanks!

Reply:
@mikemee No biggie. I'd rather have everything in the new version on jetpack.

That change was easy to implement in my code without requiring a new library.

Reply:
@ethauvin We haven't been able reproduce the bug that this last change is fixing. Do you have an easy way to demonstrate this?  In the meantime, however, I'll merge, as our testing didn't show it caused any problems (as we expected it would not). 

But if you have a repro case, we'd still love to see it, thanks!

Reply:
@mikemee Interesting. I was able to consistently reproduce it. Here's what I did:

- Open the `about` intent in my app.
- Place the app in the background.
- Use my phone as usual for a couple hours or so, at least long enough for background activities to be put to sleep/removed from memory.
- Bring my app back to the foreground.

I'm using Android 7.1.2, which might be relevant because of the new deep sleep optimization.

The behavior is pretty well documented in various posts on StackExchange, which is where I found the solution.

Reply:
Thanks!  Oh, I never waited that long! 😱  I'll look for the SO articles too. Not worried, just doing due diligence. Many thanks.